util/upload-instance-slot-map.py

```diff
@@ -61,10 +62,16 @@ def validate_against_old_version(instances_info, region, aws_credentials=None):
     old_instances_info = get_old_instances_info(region, aws_credentials)
     for instance_type in old_instances_info:
```

Comment:
is this validation logic also comparing the values of the resources. e.g. by mistake vcpus for an instance type are changed during an update? If not we might consider using the approach suggested in the previous PR

Reply:
Added value comparison